subprojects/packagefiles/cwalk/meson.build

```diff
@@ -0,0 +1,29 @@
+project('cwalk', 'c',

```

Comment:
Upstream already has a `meson.build` file. Could this not use that one instead?

Reply:
It needs a fix that has just merged. We will be able to use upstream meson file in the next version.